Doc/reference/datamodel.rst

```diff
@@ -510,6 +510,10 @@ Callable types
       | :attr:`__closure__`     | ``None`` or a tuple of cells  | Read-only |
       |                         | that contain bindings for the |           |
       |                         | function's free variables.    |           |
+      |                         | The ``cell_contents``         |           |
+      |                         | attribute can be used for     |           |
+      |                         | used for reading and writing  |           |
```

Comment:
It is kind of awkward trying to fit it in the table, I'll try to put it somewhere below.